pkgs/tools/graphics/pfstools/default.nix

```diff
@@ -1,24 +1,29 @@
-{stdenv, fetchurl, libtiff, openexr, imagemagick, libjpeg, qt4, mesa,
-freeglut, bzip2, libX11, libpng, expat, pkgconfig }:
+{ stdenv, fetchurl, cmake, pkgconfig, openexr, ilmbase, zlib, imagemagick, mesa, freeglut, fftwFloat, fftw, gsl, libexif, perl, opencv, qt4 }:
 
 stdenv.mkDerivation rec {
-  name = "pfstools-1.8.5";
+  name = "pfstools";
+  version = "2.0.5";
 
   src = fetchurl {
-    url = "mirror://sourceforge/pfstools/${name}.tar.gz";
-    sha256 = "01kk2r8afrb3vrhm8abfjdhhan97lzpapc4n8w1mpdp3kv9miy9c";
+    url = "http://downloads.sourceforge.net/project/pfstools/pfstools/${version}/${name}-${version}.tgz";
```

Comment:
Please change back to using a mirror url; downloads.sourceforge.net is now the first mirror tried, if that's a concern.